lib/TPP/IR/StructuredOpMatcher.cpp

```diff
@@ -201,6 +201,80 @@ bool tpp::structured_match::WithSingleOpImpl::withSingleOpImpl(
   return true;
 }
 
+// FIXME: This is a generalization of the method above and will eventually replace
+// the matcher for both no-op (yield) and one op (add, max).
+bool tpp::structured_match::withOpChainImpl(
+    Region *region, Operation *op, SmallVectorImpl<Value> *capturedOperands,
+    SmallVectorImpl<TypeCheckFunc>& typeChecks) {
+
+  // Numer of ops includes yield
```

Comment:
```suggestion
  // Number of ops includes yield
```

Reply:
Does it really include yield if it requires `numOps + 1` later?

Reply:
No it did not. I think it did in a previous iteration and I didn't update the comment. Since it's only used once, I changed it to include again.